app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md

```diff
@@ -22,8 +22,9 @@ keywords:
   - apply
   - how to apply
   - applying for teacher training
-promo_content:
-    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+cta_adviser:
+  adviser:
+    text: An adviser with years of teaching experience can help you understand why your application was unsuccessful and what you can do next. Chat by phone, text or email, as little or as often as you need.
```

Comment:
I'm not sure they can help with understanding why the application was unsuccessful - it depends how much information the provider has supplied?

Could we make this a bit more positive re supporting you to reapply / maximise your chances of success?